busy_beaver/apps/github_summary/workflow.py

```diff
@@ -0,0 +1,92 @@
+import logging
+import uuid
+from urllib.parse import urlencode
+
+from busy_beaver.config import GITHUB_CLIENT_ID, GITHUB_REDIRECT_URI, MEETUP_GROUP_NAME
+from busy_beaver.extensions import db
+from busy_beaver.models import User
+from busy_beaver.toolbox import EventEmitter
+
+#Setup logging
```

Comment:
Could you please remove superfluous comments?

Reply:
Resolved in this [commit](https://github.com/busy-beaver-dev/busy-beaver/pull/173/commits/22c2a56ef83bb77c842682a2cf105ecfdcecf00d)